Frameworks/Foundation/NSDecimalNumber.mm

```diff
@@ -14,296 +14,360 @@
 //
 //******************************************************************************
 
-#import "Starboard.h"
-#import "StubReturn.h"
-#include <Foundation/Foundation.h>
+#import <Starboard.h>
+#import <StubReturn.h>
+#import <Foundation/Foundation.h>
+#import <Foundation/NSDecimal.h>
+#import <Foundation/NSDecimalNumberHandler.h>
+#import <vector>
+#import <algorithm>
+#import <limits>
 
 NSString* const NSDecimalNumberExactnessException = @"NSDecimalNumberExactnessException";
 NSString* const NSDecimalNumberOverflowException = @"NSDecimalNumberOverflowException";
 NSString* const NSDecimalNumberUnderflowException = @"NSDecimalNumberUnderflowException";
 NSString* const NSDecimalNumberDivideByZeroException = @"NSDecimalNumberDivideByZeroException";
 
+const int _MAX_EXP = 127;
```

Comment:
this is internal and used only once, i'm going to leave it as as for now, as opposed to having an internal header.

---
In reply to: [81209839](https://github.com/Microsoft/WinObjC/pull/1032#discussion_r81209839) [](ancestors = 81209839)